Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs

```diff
@@ -163,6 +163,33 @@ public IEnumerator BoundsControlInstantiate()
             yield return null;
         }
 
+        /// <summary>
+        /// Verify correct collider attachment for handles after bounds control instantiation
+        /// </summary>
+        [UnityTest]
+        public IEnumerator HandleColliderInstantiation([ValueSource("handleTestData")] HandleTestData testData)
+        {
+            BoundsControl boundsControl = InstantiateSceneAndDefaultBoundsControl();
+            yield return VerifyInitialBoundsCorrect(boundsControl);
+            Assert.IsNotNull(boundsControl);
```

Comment:
Should we add a msg if assert fail, like the other asserts on the test?

Reply:
i think it should be clear, but just in case i'll add the message as well. can't harm :)